doc_build/library/integrator.rst

```diff
@@ -1,4 +1,142 @@
 :todo:
 
+###########
 Integrators
 ###########
+
+Integrators are intended as the main way for standard users to interact with ZuNIS.
```

Comment:
Most other pages of the doc call it "ZüNIS", wheras this page refers to "ZuNIS" three times.